.github/workflows/build-and-push.yml

```diff
@@ -20,29 +21,64 @@ jobs:
       run: sudo ./.github/install-etcd.sh
     - name: Build with Maven
       run: mvn -B package --file pom.xml
+
   build:
     needs: test
     runs-on: ubuntu-latest
+
     env:
-      IMAGE_NAME: kserve/modelmesh
+      IMAGE_NAME: modelmesh
+
     steps:
-    - uses: actions/checkout@v2
-    - name: Build and push runtime image
-      run: |
-        GIT_COMMIT=$(git rev-parse HEAD)
-        BUILD_ID=$(date '+%Y%m%d')-$(git rev-parse HEAD | cut -c -5)
+      - uses: actions/checkout@v2
+
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to DockerHub
+        if: github.event_name == 'push'
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_ACCESS_TOKEN }}
+
+      - name: export version variable
+        run: |
+          GIT_COMMIT=$(git rev-parse HEAD)
+          BUILD_ID=$(date '+%Y%m%d')-$(git rev-parse HEAD | cut -c -5)
+          
+          IMAGE_ID=kserve/$IMAGE_NAME
+
+          # Change all uppercase to lowercase
+          IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
```

Comment:
Could this ever not be lower case -- given that both `IMAGE_ID` and `IMAGE_NAME` are explicitly defined in the workflow template?

Reply:
it's feature parity with kserve/kserve workflow files https://github.com/kserve/kserve/blob/v0.10.0/.github/workflows/kserve-controller-docker-publish.yml#L66

Reply:
same here, fine by me to remove

Reply:
lets remove it -- less is more here :-)